tools/run_tests/xds_k8s_test_driver/framework/rpc/grpc_channelz.py

```diff
@@ -100,6 +100,22 @@ def find_server_socket_matching_client(server_sockets: Iterator[Socket],
                 return server_socket
         return None
 
+    @staticmethod
+    def channel_repr(channel: Channel) -> str:
+        result = f'<Channel channel_id={channel.ref.channel_id}'
```

Comment:
Do we always have those fields?

Reply:
Yes, they come from the proto:
https://github.com/grpc/grpc/blob/7564664f077cdaa720451633107352cb39e233e4/src/proto/grpc/channelz/channelz.proto#L36-L41

If they're not populated, there's something very wrong with channelz - so IMO `AttributeError` failure would be acceptable behavior.